\subsection{Molecular Events in Primary and Metastatic Colorectal Carcinoma: A Review}

  Recognition that histologically identical tumours may have drastically
  different prognosis and/or response to treat- ment prompted the theory that,
  rather than a single malignancy, CRC is a heterogenous, multifactorial
  disease [5, 6].

  Cancer develops through multiple and sequential genetic alterations [3, 9],
  and some patients may have synchronous alterations in two or three different
  pathways [10]. Through clonal selections, the cancer cell “chooses” the
  genetic alterations most con- ducive to growth through proliferation of cells
  that possess the desired qualities with apoptosis of those that do not [2].

  The first model of colorectal tumorigenesis put forward by Fearon and Vogelstein outlined a four-step sequential path- way for the development of cancer, in which
    Step 1: APC inactivation causes adenoma development,
    Step 2: KRAS mutations promote adenomatous growth,
    Step 3: genetic alterations of chromosome 18q allowed pro- gression with biallelic loss,
    Step 4: p53 inactivation triggers the final transition to carci- noma [11, 12].

  Recent insights suggest this pathway, now over twenty years old, requires
  refinement to include new findings [13]. Further, this sequence is thought to
  occur in only 60\\% of cases [14]. In the process of elucidating the true
  pathogenesis of CRC, controversy has emerged between those postulating that
  genomic instability is necessary to elucidate the multiple mutations present
  in CRC and those disagreeing, hypothesiz- ing instead that cells continuously
  produce genetic changes with those that confer a survival advantage being
  selected through clonal expansion [12].

  Carcinogenesis is now viewed as an imbalance between mutation development and
  cell-cycle control mechanisms. When the cell-cycle is no longer capable of
  controlling the mutation rate, it is referred to as “genomic instability.”
  Three separate pathways have been identified that contribute to this
  imbalance:
    (1) chromosomal instability (CIN),
    (2) microsatellite instability (MSI),
    (3) CpG island methylator phenotype (CIMP).

  The chromosomal instability (CIN) pathway, also known as the suppressor
  pathway, is the most common type of genomic instability [8], encompassing
  50–85\% of CRCs [5, 17]. This pathway is characterized by karyotypic
  variability resulting from gains and/or losses of whole/portions of
  chromosomes [13]. Various mechanisms that contribute to CIN have been
  identified and categorized to include (a) sequence changes, (b) chromosome
  num- ber alterations, (c) chromosome rearrangements, and (d) gene
  amplification [18]. Additional changes identified in- clude chromosomal
  segregation defects/microtubule dys- function, abnormal centrosome number,
  telomere dysfunc- tion/telomerase overexpression, DNA damage, and loss of
  heterozygosity (LOH) [13, 18, 19].

  There remains no clear evidence to discern whether CIN is a cause or a
  consequence of malignancy [13, 20]. It is clear, however, that CRC with CIN
  confers a poor survival regardless of ethnicity, tumour location, and
  treatment with 5-FU [23].

  Microsatellite instability (MSI) is detected in 15\\% of CRCs and arises when
  microsatellites become abnormally long or short due to gain/loss of re- peated
  units [24]. A microsatellite is a stretch of DNA containing a pattern of 1–5
  nucleotides in length with tandem repeats [22, 24]. Microsatellites are found
  abundantly throughout the genome and are unique in uniform and length within a
  tissue of one individual [12]. A minimum of 500,000 microsatellites are
  estimated within the genome, occurring in the intron, untranslated terminal
  regions, and the coding exon itself [26]. Microsatellites may be classified as
  monomorphic (the same number of repeats in all individuals) or polymorphic
  (varied number of repeats among individuals) [26]. Elonga- tion or shortening
  of the microsatellite is primarily due to inactivation of DNA mismatch repair
  (MMR) genes, which are responsible for correcting base-base DNA replication
  errors. At regions of short repeats within the genome, such as satellites, DNA
  polymerase is particularly susceptible to making mistakes; therefore, when MMR
  is inactivated and cannot correct these mistakes, MSI is the result [3]. This
  inactivation may be genetic or acquired. These tumours usually are not
  associated with mutations in KRAS or TP53; however, genes such as TGFβRII,
  EGFR, and BAX, which contain simple repeats, are often mutated in these
  tumours [5].

  The newest of the three genomic instability pathways, the CpG island methy-
  lator phenotype (CIMP) was originally grouped together with MSI. CpG islands
  are regions within the genome rich in CpG (Cytosine-phosphate-guanine)
  dinucleotides where cytosine DNA methylation does not covalently modify [2].
  These islands are especially common in promoter sequences, found in over half
  of them [2]. In normal tissue cytosine methylation is common outside of the
  exons [8]. Methylation naturally increases with age, injury, and in patients
  with chronic inflammation [12]. The epigenetic modification of methylation is
  well recog- nized as a crucial event in altering gene expression associated
  with carcinogenesis and is more frequent in cancer than genetic changes [5].
  Methylation of promoter CpG islands occurs in all tissue types in
  carcinogenesis [34]. Methylation leads to transcriptional silencing of genes
  involved in tumour suppression, cell cycle control, DNA repair, apoptosis, and
  invasion [35]. CIMP positivity is found in 35–40\\% of CRCs and has
  additionally been identified in adenomas [17]. It is postulated DNA
  methylation may be altered in normal colorectal mucosa, predisposing the
  affected tissue to further dysplastic changes. It is the second most common
  cause of sporadic CRC [3]. Through hypermethylation of histone CpG islands,
  the chromatin closes into a compact structure such that the gene promoter is
  inaccessible to transcription factors, thereby inactivating gene
  transcription.

  4.1. Mutational Inactivation of Tumour-Suppressor Genes.
    Tumour suppressor genes code proteins that act to limit growth and
    proliferation, the cell cycle, motility, and inva- sion in normal human tissues
    [8]. In carcinogenic transfor- mation, tumour growth is often facilitated by
    inactivation of these genes through deletions, mutations, promoter methyl-
    ation, or mutation of one allele with loss of the other [12, 39]. Several key
    players in the carcinogenetic process have been identified and are well
    elucidated in the literature. These genes include APC, TP53, and TGF-β.

  4.2. Activation of Oncogene Pathways. Oncogenes are genes with the potential
    to cause cancer. They encode growth factors, growth factor receptors,
    signalling molecules, reg- ulators of the cell cycle, and additional factors
    implicated in cellular proliferation and survival. When these genes are
    mutated, results may include overactive gene products, am- plifications
    altering copy number, alterations that affect pro- moter function or modified
    interactions that cause transcrip- tion/epigenetic modifications.

  4.2.1.RASandBRAF. TheRAS-RAF-MAPKpathwaybegins with a mitogen (such as EGF)
    binding to the membrane re- ceptor (such as EGFR), which allows the GTPase Ras
    to exchange its GDP for a GTP, activating MAP3K (Raf) which activates MAP2K
    which activates MAPK. MAPK then activates transcription factors that express
    proteins with a role in cellular proliferation, differentiation, and survival
    [45]. A key, and well-studied, component of this cascade is the Ras family,
    comprised of three members: KRAS, HRAS, and NRAS. These isoforms are located on
    the inner surface of the plasma membrane [45]. A common target of somatic
    mutations, especially at codons 12 (82–87\%), 13 (13–18\%), and 61, KRAS has
    been implicated in many human cancers [46]. KRAS mutations have been reported in
    40\% of CRCs and contribute to the development of colorectal adenomas and
    hyperplastic polyps [2]. These mutations are usually single nucleotide point
    mutations that lock the enzyme bound to ATP, by inhibiting its GTPase activities
    thus upreg- ulating the Ras function [13, 22]. It, therefore, affects downstream
    signalling cascades including MAPK and PI3K. Early KRAS mutations have been
    identified in left-sided hyperplastic polyps [10]. This mutation is more common
    in polypoid lesions than nonpolypoid [6]. KRAS mutations are associated with a
    worse prognosis, in part due to the overexpression of KRAS contributing to
    metastases through increasing the production of protease to degrade the ex-
    tracellular matrix. However, the prognostic role of KRAS mutations remains
    largely ill understood and further studies are required. Attempts have been made
    at targeting KRAS for cancer treatment including inhibition of protein
    expression through antisense oligonucleotides and with blockage of
    posttranslational modifications to inhibit downstream effec- tors [5]. KRAS
    mutations have recently gained interest as a negative predictive factor for
    anti-EGFR therapy response. Blocking EGFR will have no effect if KRAS is mutated
    as it functions downstream of EGF receptors. Thus, a KRAS mutation allows
    continual activation of the downstream pathway, thus negating the effects of the
    drug [41]. As such, anti-EGFR drugs (Section 7.1.2) are not recommended in
    KRAS-mutated tumours. In this context, it has been suggested that all patients
    with CRC under consideration for anti-EGFRs should be tested for KRAS mutation
    status prior to treatment initiation [16, 41]. The Raf family includes three
    members: ARAF, RAF1, and BRAF. When activated, these serine/threonine kinases
    activate MEK1 and MEK2 which phosphorylate ERK1 and ERK2. The ERKs continue the
    cascade by phosphorylating cytosolic and nuclear substrates such as JUN and ELK1
    that regulate a wide spectrum of enzymes such as cyclin D1 [13]. Similar to
    KRAS, BRAF mutations render it continually active, in over 80\% of CRCs by
    substitution of thymine to adenine at nucleotide 1799 that results in a
    substitution of valine to glutamic acid [5]. These point mutations make BRAF an
    attractive marker for analysis, as they are present in at least 80\% of mutants
    [22]. Such mutations are more frequent in MSI tumours and are reported in 5–15\%
    of CRCs [5]. Mutations in BRAF and KRAS are mutually ex- clusive as they are
    intimately connected in the RAS-RAF- MAPK pathway [45]. In the rare instance of
    concomitant mutations, they confer a synergistic effect and increase tum- our
    progression [5]. BRAF mutations confer a worse clinical outcome and thus the
    need for adjuvant therapy [5]. Mutations are asso- ciated with a shorter
    progression free and overall survival [45]. Though controversial, some studies
    have found that these adverse clinical effects of BRAF are negated in CIMP+
    tumours, suggesting the poor prognosis is not attributable to the BRAF mutation
    itself, but is probably attributable to the genetic pathway in which it occurs
    [5]. Similar to KRAS, BRAF mutations have also been implicated in anti-EGFR
    resistance. Approximately 60\% of KRAS wild-type metastatic CRC (mCRC) are
    unresponsive to these drugs, and it is hypothesized that BRAF mutations may
    confer some of this resistance [41]. As such, BRAF mutation status may also be
    assessed to determine patients resistant to anti-EGFR therapy.

  4.2.2. Phosphatidylinositol 3-Kinase (PI3K). The PI3K-Akt begins with
    activation of PI3K, which can occur in three ways: (1) a growth factor binds
    to IGF-1 receptor in the cell membrane, causing dimerization and
    autophosphorylation of the receptor, IRS-1 then binds to the receptor and acts
    as a binding site and activator of PI3K; (2) a growth factor binds to a
    receptor tyrosine kinase embedded in the membrane, again causing dimerization
    and autophosphorylation, the PI3K then binds directly to the receptor and is
    activated; (3) the GTPase Ras (as seen above) may bind PI3K and activates it.
    Once PI3K is activated, it detaches and phosphorylates PIP2 that is a
    component of the membrane, transforming it to PIP3. PIP3 then activates Akt, a
    proto-oncogene with functions including growth promotion, proliferation, and
    apoptosis inhibition [5, 22]. The system is restored by PTEN, which
    dephosphorylates PIP3 and inhibits Akt signalling. PI3Ks are a family of
    enzymes divided into three classes. Of interest in CRC is the class 1A PI3Ks,
    which are composed of one catalytic subunit (either p110α, p110β, or p110δ)
    and one regulatory subunit (p85α, p85β or p86γ) [22]. The catalytic subunit
    p110α, which is encoded by the protein PIK3CA, has been of particular
    attention as it is believed to play a significant role in cancer
    progression. Its mutation has been detected in approximately a third of CRCs
    [8]. These gain-of-function mutations cause increased Akt signalling even
    without the presence of growth factors [45]. Clinically, the prognostic role
    of PI3KCA is under investigation, and it is suspected to confer a poor
    outcome [5]. It has also been suggested to play a role in resistance to
    anti-EGFR treatment [45]. Due to its inhibitory effect on Akt, the
    phosphatase and tensin homolog (PTEN) acts as a tumour suppressor gene in
    this pathway. In CRC, the PTEN gene may be inactivated by somatic mutations,
    allelic loss or hypermethylation of the enhancer region [45]. Mutation of
    PTEN is a later event in carcinogenesis that is correlated with advanced and
    metastatic tumours. Though it is clear that PTEN mutations in stage II CRC
    is a poor prognostic marker, its role in other stages of CRC remains
    uncertain [5]. There remains a discrepancy as to the exact role of PTEN in
    anti-EGFR resistance [45].


  MicroRNA (miRNA) is short RNA 8–25 nucleotides in length that binds to mRNA to
    control translation of com- plementary genes [47]. Over 1,000 miRNA sequences
    have been identified, each controlling hundreds of genes for a total of over
    5,300 genes, 30\% of the human genome [48]. These short RNAs play a regulatory
    role in development, cellular differentiation, proliferation, and apoptosis. In
    this context, miRNA dysregulation is suggested to play a role in carcino-
    genesis when their mRNA targets are tumour suppressor genes or oncogenes through
    silencing and overexpression respectively [47]. Half miRNAs are located at the
    breakpoints of chromosomes and, therefore, at a higher risk of dysregu- lation
    [48]. Whether or not the microenvironment directly affects miRNA dysregulation
    remains unclear [49]. Mature miRNA conducive to tumour growth may be upregulated
    through transcriptional activation and/or amplification of the miRNA encoding
    gene and those unfavourable to growth are silenced by deletion or epigenetic
    modifications [47]. Overexpression of miR-31, -183, -17-5, -18a, -20a, and -92
    and underexpression of miR-143 and -145 are common in CRC [50]. It remains
    unclear, however, which miRNA changes are causative and which are a result of
    carcinogenesis [2].

  6.1.HistoneModification. Histonesareproteinsthatpackage cellular DNA into
    nucleosomes and play a role in gene reg- ulation. Covalent modifications,
    including acetylation, methylation, phosphorylation, and ubiquitinylation of
    these proteins can cause dense inactive heterochromatin to open to euchromatin
    and vice versa. Such modifications are reversible, usually occurring at the N-
    and C-terminal re- gions [51]. Hypoacetylation and hypermethylation are char-
    acteristic of transcriptionally repressed chromatin regions. Mutations in
    histones are most common at lysine and arginine residues [33]. The mutually
    exclusive modifications that have been identified in CRC include deacetylation
    and methylation of lysine 9 in histone H3. If acetylation occurs at this
    position, the gene is expressed whereas if it is methylated, the gene is
    silenced. It is suggested that a universal marker for malignant transformation
    is the loss of monoacetylation from Lys 16 and trimethylation at Lys 20 on
    histone H4 [51].

6.2. DNA Methylation. The process of hypermethylation of CpG islands is
  discussed above (Section3.3); there- fore, this section will discuss the process
  of global DNA hypomethylation, a process not as well understood. Over 40\% of
  human DNA contains short interspersed transpos- able elements (SINEs) and long
  interspersed transposable elements (LINEs) that are normally methylated but
  become hypomethylated in CRC development [35]. Hypomethyla- tion most commonly
  occurs at repetitive sequences such as satellites or pericentromeric regions
  [51]. This epigenetic change increases chromosomal susceptibility to breakage,
  thus creating genomic instability, reactivating retrotrans- posons that disrupt
  gene structure and function, or acti- vating oncogenes such as the S100A4
  metastasis-associated gene in CRC [51]. These changes are believed to occur
  early in carcinogenesis, as hypomethylation is detected in benign polyps but is
  not changed once they become malignant [35].

\subsection{Systematic genomic identification of colorectal cancer genes delineating advanced from early clinical stage and metastasis}

  Identifying the genetic and genomic basis of CRC has sig- nificant clinical
  implications. Our understanding of CRC requires identification of the critical
  “driver” genes that are fundamentally important for CRC development unlike
  “passenger” genetic aberrations that have no functional relevance to cancer
  biology [2]. Previous genetic and gen- omic studies of CRC have identified
  many of the critical drivers that are important to CRC development [3-7]. For
  example, the cancer genes APC, KRAS and TP53 have a high frequency of genetic
  aberrations in CRC and are known to play an essential role in CRC development
  [8]. A number of other cancer genes have been identified in CRC and cluster in
  several biological pathways including those responsible for Wnt signaling [9],
  RAS/RAF path- way [10] and transforming growth factor β (TGF-β) signaling
  [11].

  Currently, the metastatic status of CRC is assessed via clinical staging which
  dictates the choice of therapy and remains the best prognostic indicator for
  individual CRC patients [12]. Clinical stage is determined by the TNM
  criteria, where T is assigned by extent of tumor inva- sion, N represents the
  number of lymph nodes with metastatic cancer and M represents the presence of
  metastatic cancer in other organs outside of the colon and lymph nodes.

  ith the advent of gen- omic cancer medicine, there is increasing interest in
  identifying the specific CRC genetic aberrations and re- lated cancer genes
  that define advanced clinical stage. Identification of these genetic
  aberrations and their cor- responding cancer genes may illuminate the
  underlying genetics of advanced clinical stage CRC as well as have relevance
  in the prognostic assessment.


\subsection{A Genetic Model for Colorectal Tumorigenesis}

  Tumorigenesis = multistep process
  malignant CRCs arise from preexisting benign tumors (adenomas)

  1. CRCs appear to arise as a result of the mutational activation of oncogenes
      coupled with the mutational inactivation of tumor suppressor genes
  2. mutations in at least 4-5 genes are required for the formation of a
      malignant tumor. Fewer changes suffice for benign tumorigenesis
  3. Although the genetic alterations often occur according to a preferred
      sequence, the total accumulation of changes, rather than their order with
      respect to one another, is responsible for determining the tumor's properties
  4. in some cases, mutant tumor suppressor genes appear to exert a phenotypic
      efect even when present in the heterozygous state

  The clonal nature of colonic neoplasia:
